services/discovery/v1-query-builder.js

```diff
@@ -45,13 +59,13 @@ module.exports = function(RED) {
                                          sApikey ? sApikey : req.query.key,
                                          req.query.endpoint ? req.query.endpoint : sEndpoint);
 
-    discovery.listEnvironments({}, function(err, response) {
-      if (err) {
+    discovery.listEnvironments({})
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 spaces but found 4.](https://app.codacy.com/manual/BetaWorks-NodeRED-Watson/node-red-node-watson/pullRequest?prid=4846228)